snapshotTestContent/DuplicateGCRoutes/current_snapshots/snapshot_202_202.json

```diff
@@ -0,0 +1,440 @@
+                      "encoding": "utf-8"
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        }
+      },
+      {
+        "mode": "100644",
+        "path": ".metadata",
+        "type": "Blob",
+        "value": {
+          "contents": "{\"createContainerRuntimeVersion\":\"0.58.2002\",\"createContainerTimestamp\":1650582895090,\"summaryCount\":2,\"summaryFormatVersion\":1,\"gcFeature\":1,\"message\":{\"clientId\":null,\"clientSequenceNumber\":-1,\"minimumSequenceNumber\":200,\"referenceSequenceNumber\":-1,\"sequenceNumber\":202,\"timestamp\":1651091109040,\"type\":\"join\"}}",
+          "encoding": "utf-8"
+        }
+      },
+      {
+        "mode": "040000",
+        "path": "gc",
+        "type": "Tree",
+        "value": {
+          "entries": [
+            {
+              "mode": "100644",
+              "path": "__gc_root",
+              "type": "Blob",
+              "value": {
+                "contents": "{\"gcNodes\":{\"/defaultComponent/root\":{\"outboundRoutes\":[\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4\",\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a\",\"/defaultComponent\"]},\"/defaultComponent\":{\"outboundRoutes\":[\"/defaultComponent/root\"]},\"/atMentions/instances\":{\"outboundRoutes\":[\"/atMentions\"]},\"/atMentions\":{\"outboundRoutes\":[\"/atMentions/instances\"]},\"/_scheduler/517535cd-da40-40eb-a9d1-893a9bfa3de7\":{\"outboundRoutes\":[\"/_scheduler\"]},\"/_scheduler/root\":{\"outboundRoutes\":[\"/_scheduler/517535cd-da40-40eb-a9d1-893a9bfa3de7\",\"/_scheduler\"]},\"/_scheduler\":{\"outboundRoutes\":[\"/_scheduler/root\",\"/_scheduler/517535cd-da40-40eb-a9d1-893a9bfa3de7\"]},\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4/ecf6d2da-e076-4e29-a1c5-d0c7c71b0657\":{\"outboundRoutes\":[\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4\"]},\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4/root\":{\"outboundRoutes\":[\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4/ecf6d2da-e076-4e29-a1c5-d0c7c71b0657\",\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4\"]},\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4\":{\"outboundRoutes\":[\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4/root\",\"/6618c4ff-7a76-4f58-b8e9-275a20545fb4/ecf6d2da-e076-4e29-a1c5-d0c7c71b0657\"]},\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a/root\":{\"outboundRoutes\":[\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a/2fe55675-867b-45ae-82fe-30768b84037a\",\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a\"]},\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a/2fe55675-867b-45ae-82fe-30768b84037a\":{\"outboundRoutes\":[\"/atMentions/50ea661e-1069-402e-867d-69282da5068a\",\"/atMentions/3471efaa-1a8a-4878-b247-99da73c4e790\",\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a\"]},\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a\":{\"outboundRoutes\":[\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a/2fe55675-867b-45ae-82fe-30768b84037a\",\"/4b9f4e0c-5d7c-4ad5-ab32-4e1db683c28a/root\"]},\"/\":{\"outboundRoutes\":[\"/defaultComponent\",\"/atMentions\",\"/_scheduler\"]}}}",
```

Comment:
I believe you re-generated these snapshots with the fix in https://github.com/microsoft/FluidFramework/pull/10016. The duplicate routes are gone from the GC data because of the fix :-). You have two options:
1. Generate the snapshots without the fix, merge the PR and update FF repo to point to the new commit. Then, update the snapshot (npm run test:update) in the repo that has the fix to update these snapshots and follow the same steps to check these in.
2. Generate the snapshots without the fix, get these snapshots in the repo with the fix and run `npm run test:update` to update the snapshots. Push these updated snapshots, merge them and then update your PR with the fix to point to this new commit.